metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -288,13 +288,27 @@ class PowerBiProfilingConfig(ConfigModel):
     )
 
 
+class PowerBiEnvironment(str, Enum):
```

Comment:
Probably don't need both `str` and `Enum`. `datahub.configuration.common.ConfigEnum` would work well here.